pkg/ottl/ottlfuncs/README.md

```diff
@@ -890,6 +891,25 @@ Examples:
 
 - `Hours(Duration("1h"))`
 
+### HTMLStrip
```

Comment:
I'm wondering if we should rename it something like `stripHTML`, I personally prefer the `<action><subject>` naming pattern, similar to the existing XML functions.

Reply:
I totally agree